docs/lcb/notes/feature-hex_bin_literals.md

```diff
@@ -0,0 +1,20 @@
+# LiveCode Builder Language
+## Numeric literal improvements
+
+* Base 2 (binary) integer literals can now be specified by using a "0b" prefix, e.g.
+
+      0b0000
+      0b1010
```

Comment:
Due to a quirk of Markdown, I believe this code block and the code block in the following bullet point will only get successfully parsed as code blocks if you wrap the first paragraph of bullet point, e.g.:

```markdown
* Base 2 (binary) integer literals can now be specified by using a "0b"
  prefix, e.g.

      0b0000
      0b1010
```

Yes, Markdown is horrible.